frontend/packages/core/src/tests/landing.test.tsx

```diff
@@ -1,23 +1,22 @@
 import React from "react";
 import { MemoryRouter } from "react-router-dom";
-import { shallow } from "enzyme";
+import { render } from "@testing-library/react";
+
+import "@testing-library/jest-dom";
 
 import * as appContext from "../Contexts/app-context";
 import Landing from "../landing";
 
-describe("Landing component", () => {
-  let component;
+beforeAll(() => {
```

Comment:
Is the beforeAll required here? Since we're only mocking it once it should be fine to leave it as the base level right?